backend/mlarchive/tests/archive/views.py

```diff
@@ -82,14 +81,11 @@ def test_get_this_next_periods(static_list):
     assert get_this_next_periods(time_period) == (
         datetime.datetime(2017, 1, 1, tzinfo=timezone.utc),
         datetime.datetime(2018, 1, 1, tzinfo=timezone.utc))
-
-
-@pytest.mark.django_db(transaction=True)
-def test_add_one_month():
```

Comment:
You removed the test - is there an `add_one_month()` method to be removed as well?

Reply:
Ooops. Removed now.